tests/client/ui/components/TestLoadingIndicator.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Simulate, renderIntoDocument } from 'react-addons-test-utils';
+
+import LoadingIndicator from 'ui/components/LoadingIndicator';
+
+
+describe('<LoadingIndicator />', () => {
+  function render(props) {
+    return renderIntoDocument(<LoadingIndicator {...props} />);
+  }
+
+  it('resets the animation after it ends', (done) => {
+    const root = render({ timeToReset: 5 });
+    const iconNode = ReactDOM.findDOMNode(root.loadingIcon);
+
+    assert.isTrue(root.state.animating);
+
+    Simulate.animationEnd(iconNode);
+
+    assert.isFalse(root.state.animating);
+
+    setTimeout(() => {
+      assert.isTrue(root.state.animating);
+      done();
+    }, 10);
```

Comment:
These tests shouldn't be needed if this is done in CSS but sinon provides [fake timers](http://sinonjs.org/docs/#clock) that can be used to test code with `setTimeout()` synchronously if we need it elsewhere.

We're currently using them in [TestHoverIntent.js](https://github.com/mozilla/addons-frontend/blob/89ff059ead85a4c813d018388ed093d4a1b4eaf4/tests/client/core/components/TestHoverIntent.js#L51).